CONTRIBUTING.md

```diff
@@ -22,8 +22,7 @@ To revert back to your original version of Tribler, [download the installer agai
     * The version of Tribler that you are using, if you are running from a branch, branch name and commit ID.
     * The OS and version you are running.
     * Step by step instructions to reproduce the issue in case it's a bug.
-    * Attach Tribler's log file found in `%APPDATA%` on Windows or on
-      `~/.Tribler/` on Linux/OSX (have a look at its contents before posting it
+    * Attach Tribler's log file found in `%APPDATA%` on Windows, `~/.Tribler/` on Linux or the crash log found in `~/Library/Logs/DiagnosticReports` on OS X (have a look at its contents before posting it
```

Comment:
I just created a crash in the Python interpreter and the full info is written to the files found in `~/Library/Logs/DiagnosticReports`. For each crash, a file is generated with the stack trace. The name of the file indicates the program that has crashed (for instance, I have a `Tribler_2016-01-10-205136_MacBook-Pro-van-Martijn.crash` in this directory).

When I created the crash, the syslog only contained a line like `crash log written to ...` where `...` is the location of the crash log file.